src/screens/DiscoveryScreenV2/fragment/UserFragmentCommunity.js

```diff
@@ -230,7 +230,7 @@ const UsersFragmentCommunity = ({
         data={data || []}
         renderItem={renderItem}
         keyExtractor={(item, index) => index.toString()}
-        onEndReached={() => fetchData()}
+        onEndReached={() => (data.length > 0 ? fetchData() : null)}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change made here is a good one. It adds a check to ensure that `fetchData()` is only called when there are items in the `data` array. This can prevent unnecessary network requests and improve performance.
```diff
- onEndReached={() => fetchData()}
+ onEndReached={() => (data.length > 0 ? fetchData() : null)}
```
However, it's worth noting that if `data` is `null` or `undefined`, this line will throw an error because you're trying to access `length` of something that's not an object. To avoid potential errors, you should add a check for `data` before accessing its `length`.
```diff
- onEndReached={() => (data.length > 0 ? fetchData() : null)}
+ onEndReached={() => (data && data.length > 0 ? fetchData() : null)}
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->